features/kafka/producer/src/main/java/org/opennms/features/kafka/producer/OpennmsKafkaProducer.java

```diff
@@ -96,6 +100,11 @@
     private final AlarmEqualityChecker alarmEqualityChecker =
             AlarmEqualityChecker.with(AlarmEqualityChecker.Exclusions::defaultExclusions);
 
+    // TODO: Determine an appropriate queue capacity
+    private final BlockingQueue<KafkaRecord> kafkaSendQueue =
+            new LinkedBlockingQueue<>(1000);
+    private final Executor kafkaSendQueueExecutor = Executors.newSingleThreadExecutor();
```

Comment:
We should use the constructor that takes a ThreadFactory, so that we can set the name of the thread - which is useful for identifying it in a thread dump.